backend/internal/storage/postgres/schema/review.go

```diff
@@ -0,0 +1,66 @@
+package user
+
+import (
+	"context"
+	"platnm/internal/models"
+
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type ReviewRepository struct {
+	db *pgxpool.Pool
+}
+
+func (r *ReviewRepository) GetReviewsByUserID(ctx context.Context, id string) ([]*models.Review, error) {
+
+	rows, err := r.db.Query(ctx, "SELECT * FROM review WHERE user_id = $1", id)
+	if err != nil {
+		print(err.Error(), "from transactions err ")
```

Comment:
remove log. also with this current setup, when a user does not exist, i believe this will return an empty review array. but if user does not exist we should prob return 404 user not found

i think u can use a join and then check: if rows come back, then user exists, else user does not exist and return `errs.NotFound`

Reply:
this is fixed now, see screenshots from postman!